code/datums/keybinding/xeno.dm

```diff
@@ -782,19 +782,31 @@
 	description = ""
 	keybind_signal = COMSIG_XENOABILITY_RAVAGE_SELECT
 
+/datum/keybinding/xeno/runner_pounce
+	name = "runner_pounce"
```

Comment:
can make this "pounce" so it doesnt reset it

Reply:
Sure. Donezo.